bin/plot_krun_results

```diff
@@ -250,19 +250,46 @@ def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
         for index, page in enumerate(pages):
             bmark, vm, mc = all_subplot_titles[index][0].split(', ')[:3]
             print 'Plotting %s: %s (%s) on page %02d of %02d.' % \
-                  (mc, bmark, vm, index + 1, len(pages)),
-            fig, export_size = draw_page(is_interactive, page, cycles_pages[index],
-                                   instr_pages[index],
-                                   all_subplot_titles[index], window_size, xlimits,
-                                   all_outliers[index], all_unique[index],
-                                   all_common[index], all_changepoints[index],
-                                   all_changepoint_means[index], median, tukey,
-                                   inset_xlimit, legend_off, core_cycles, cycles_ylimits)
-            if not is_interactive:
-                fig.set_size_inches(*export_size)
-                pdf.savefig(fig, dpi=fig.dpi, orientation='landscape',
-                            bbox_inches='tight')
-                pyplot.close()
+                  (mc, bmark, vm, index + 1, len(pages))
+
+            # strip out indicies where the benchmark crashed
+            def only_uncrashed(data):
+                if data is None:
+                    return None
+                ret = []
```

Comment:
Why do we do this by the way? `list()` is longer to type and Python has syntax for lists.